### ISSUE TITLE
need improvement in accesing weight in the Pretrained Weights section 

### ISSUE THREAD

**Issue by SophiaLi20** [openclimatefix/metnet#71]
need help in accesing weight in petrained weights section i.e., We release weights for both MetNet and MetNet-2 trained on cloud mask and satellite imagery data with the same parameters as detailed in the papers on HuggingFace Hub for [MetNet](https://huggingface.co/openclimatefix/metnet) and [MetNet-2](https://huggingface.co/openclimatefix/metnet-2)
. also there is the section for Inference API which needs imrpovement.
(GIVE-Unable to determine this model’s pipeline type) 
Below is the screesnshot of the result that is appearing 
![Screenshot (93)](https://github.com/user-attachments/assets/c8dbc567-25dd-42b7-a289-9f188b7915b0)

**Comment by jacobbieker** [comment]
Thanks for pointing this out! We can update the documentation there. 

For the inference API, we don't integrate with HuggingFace for that, as that is more geared towards LLMs I think. So that is out of the scope of these weights and repo at the moment.
